src/Core/Application/Service/ManifestValidator/ManifestValidatorRegistry.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Service\ManifestValidator;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\ManifestConfigurationInterface;
+use SprykerSdk\Sdk\Core\Application\Exception\ManifestValidatorMissingException;
+
+class ManifestValidatorRegistry
```

Comment:
Please, move it to the Infrastructure layer. Also, please, avoid using the `Service` folder, instead, use behavior-specific folder names. e.g. `Registry` for registries, `Storage` for storage classes.